lib/akismet.rb

```diff
@@ -68,8 +68,31 @@ def for_post
       extra_args
     end
 
+    def for_post_voting_comment
+      extra_args = {
+        blog: Discourse.base_url,
+        content_type: "post-voting-comment",
+        referrer: @target.custom_fields["AKISMET_REFERRER"],
+        permalink: "#{Discourse.base_url}#{@target.url}",
+        comment_author: @target.user.try(:username),
+        comment_content: post_voting_comment_content,
+        comment_author_url: @target.user&.user_profile&.website,
+        user_ip: @target.custom_fields["AKISMET_IP_ADDRESS"],
+        user_agent: @target.custom_fields["AKISMET_USER_AGENT"],
+        comment_date_gmt: @target.created_at.iso8601,
+      }
+
+      # Sending the email to akismet is optional
+      if SiteSetting.akismet_transmit_email?
+        extra_args[:comment_author_email] = @target.user.try(:email)
+      end
+      @munger.call(extra_args) if @munger
+
+      extra_args
+    end
+
     def target_name
-      @target.class.to_s.downcase
+      @target.class.to_s.split(/(?=[A-Z])/).join("_").downcase
```

Comment:
Is this the same thing? If so, probably easier/clearer to use `.underscore`
![Screenshot 2024-01-10 at 2 17 38 PM](https://github.com/discourse/discourse-akismet/assets/16214023/268e3f7f-8a48-458e-831f-4db87bba57c9)